package.json

```diff
@@ -32,7 +32,6 @@
     "aes-js": "^3.1.2",
     "buffer": "^6.0.3",
     "dotenv": "^16.0.2",
-    "eas-cli": "^2.1.0",
```

Comment:
due to this recommendation:
https://github.com/expo/eas-cli#enforcing-eas-cli-version-for-your-project
>Installing eas-cli to your project dependencies is strongly discouraged because it may cause dependency conflicts that are difficult to debug.